vnext/Scripts/rnw-dependencies.ps1

```diff
@@ -216,14 +247,14 @@ $requirements = @(
     },
     @{
         Id=[CheckId]::VSUWP;
-        Name = 'Visual Studio >= 16.5 with UWP and Desktop/C++';
+        Name = 'Compilers, build tools, SDKs and Visual Studio';
         Tags = @('appDev', 'vs2019');
         Valid = CheckVS;
         Install = { InstallVS };
     },
     @{
         Id=[CheckId]::Node;
-        Name = 'NodeJS 12, 13 or 14 installed';
+        Name = 'NodeJS lts installed';
```

Comment:
Should we change `CheckNode` to be more forgiving? This doesn't seem to match up with what we're checking.

```ps1
function CheckNode {
    try {
        $v = (Get-Command node -ErrorAction Stop).Version.Major
        return $v -eq 12 -or $v -eq 13 -or $v -eq 14
    } catch {
        return $false;
    }
}
```

Reply:
E.g. any odd number of Node is not an LTS, but we allow 13. Same with 12 no longer being LTS. Maybe we should just check for Node >= 12